src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/verify/HostingFieldVerifierConsumer.java

```diff
@@ -0,0 +1,74 @@
+package io.jenkins.infra.repository_permissions_updater.hosting.verify;
+
+import io.jenkins.infra.repository_permissions_updater.hosting.HostingConfig;
+import io.jenkins.infra.repository_permissions_updater.hosting.model.HostingRequest;
+import io.jenkins.infra.repository_permissions_updater.hosting.model.VerificationMessage;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class HostingFieldVerifierConsumer implements VerifierConsumer {
+    @Override
+    public void accept(HostingRequest issue, HashSet<VerificationMessage> hostingIssues) {
+        String forkTo = issue.newRepoName();
+        String forkFrom = issue.repositoryUrl();
+        List<String> users = issue.githubUsers();
+
+        // check list of users
+        if (users.isEmpty()) {
+            hostingIssues.add(new VerificationMessage(VerificationMessage.Severity.REQUIRED, HostingConfig.RESOURCE_BUNDLE.getString("MISSING_LIST_OF_USERS")));
```

Comment:
what's the reasoning for this? we aren't likely to internationalise these so imo its just adding a layer of indirection.

Also I suspect a number of the messages are missing escaping for e.g. ' in a number of places its adding complexity

Reply:
The boiler code that may affect the readability of other developers, especially the messages, was simply moved to a file. With Intellij I can jump from string to resource bundle and back again, the complexity is reduced in this case with the code.